crates/sdk/src/sys/mock.rs

```diff
@@ -0,0 +1,62 @@
+use super::types::*;
+
+#[no_mangle]
+extern "C" fn panic(_loc: Location) -> ! {
+    panic!("Although your build was successful, it doesn't mean you're allowed to call this function in a non WASM target!")
```

Comment:
let's also simplify the message

Reply:
The funciton_name is supposed to call on WASM target. Panicking

Reply:
Does this work?

Reply:
```text
host function `panic` is only available when compiled for wasm32
```

what do you think of something like?